lib/elix/brain/redis_store.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Elix.Brain.RedisStore do
+  @moduledoc """
+  A Redis back-end for the Brain which persists memory to disk.
+  """
+
+  @behaviour Elix.Brain.Store
+  @encoded_binary_prefix "binterm:"
+
+  def start_link do
+    redis_url = System.get_env("REDIS_URL") || "redis://"
+    {:ok, _pid} = Redix.start_link(redis_url, name: :redis)
+  end
+
+  @spec set(String.t, list) :: any
+  def set(key, list) do
+    command!(["DEL", key])
+    Enum.each(list, fn (item) ->
+      command!(["RPUSH", key, encode_if_necsssary(item)])
+    end)
```

Comment:
This should probably use a `MULTI` (or whatever Redix provides for that) so that these operations are atomic and the key isn't deleted unless adding the new values also succeeds.